docs/articles/faq/README.md

```diff
@@ -32,21 +32,15 @@ This proxy injects the driver script that emulates user actions into the tested
 You can read about this in our [forum](https://testcafe-discuss.devexpress.com/t/why-not-use-selenium/47).
 Feel free to ask for more details.
 
-### What is the difference between a [paid](https://testcafe.devexpress.com) and an [open-source](https://devexpress.github.io/testcafe) TestCafe version? What is [TestCafe Studio](https://www.devexpress.com/products/testcafestudio/)?
+### What is the difference between [TestCafe Studio](https://www.devexpress.com/products/testcafestudio/) and [open-source TestCafe](https://devexpress.github.io/testcafe)? Has [TestCafe v2015.1](https://testcafe.devexpress.com) been deprecated?
 
-All three versions share the same core features:
+Both TestCafe Studio and open-source TestCafe share the same core features:
```

Comment:
(proposal)
Now it looks like Studio and TestCafe version have only these two the same features and the others, described below are different. Maybe we should include these two common features in TestCafe open source block too, because in Studio we say `Based on the open-source TestCafe, and supports its major features.` ...we can add `In addition you can...`

Reply:
Not sure. Now they are highlighted as key features of the TestCafe product family. If we move them to TestCafe, they could get lost among other features.

Maybe we should do the opposite - move a couple of key features, like Smart Assertion Query, Automatic Waiting Mechanism and Concurrent Test Execution to the first block?

What do you think? @helen-dikareva

Reply:
(discussed in private)